codecs-parent/codecs-stream/src/test/java/org/interledger/codecs/stream/frame/AsnStreamDataFrameCodecTest.java

```diff
@@ -0,0 +1,168 @@
+
+   public AsnStreamDataFrameCodecTest() {
+     super(StreamDataFrame.class);
+   }
+
+   /**
+    * The data for this test...
+    */
+   @Parameters
+   public static Collection<Object[]> data() {
+
+     return Arrays.asList(new Object[][] {
+         // streamId=0
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ZERO)
+                 .offset(UnsignedLong.valueOf(10L))
+                 .build()
+         },
+         // streamId=1
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.valueOf(10L))
+                 .build()
+         },
+         // streamId=10
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.valueOf(10L))
+                 .offset(UnsignedLong.valueOf(10L))
+                 .build()
+         },
+         // streamId=Max-1
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.MAX_VALUE.minus(UnsignedLong.ONE))
+                 .offset(UnsignedLong.valueOf(10L))
+                 .build()
+         },
+         // streamId=Max
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.MAX_VALUE)
+                 .offset(UnsignedLong.valueOf(10L))
+                 .build()
+         },
+         // Offset=0
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.ZERO)
+                 .build()
+         },
+         // Offset=1
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.ONE)
+                 .build()
+         },
+         // Offset=10
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.valueOf(10L))
+                 .build()
+         },
+         // Offset=Max-1
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.MAX_VALUE.minus(UnsignedLong.ONE))
+                 .build()
+         },
+         // Offset=Max
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.MAX_VALUE)
+                 .build()
+         },
+
+         // small data
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.MAX_VALUE)
+                 .data((byte) 1)
+                 .build()
+         },
+
+         // bigger data
+         {
+             StreamDataFrame.builder()
+                 .streamId(UnsignedLong.ONE)
+                 .offset(UnsignedLong.MAX_VALUE)
+                 .data((byte) 1, (byte) 2, (byte) 3, (byte) 4, (byte) 5, (byte) 6, (byte) 7, (byte) 8, (byte) 9,
```

Comment:
this is just copy/paste, but thinking there's a better way to generate random data of a certain size. Will change this.

Reply:
This is fixed. Also updated https://github.com/hyperledger/quilt/issues/242 with a note to revisit this test once that issue is fixed.

Reply:
`new Random(System.currentTimeMillis()).nextBytes(byteArray);` would do the trick right? or maybe use `ThreadLocalRandom`?